modules/robin/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## Unreleased
+
+### Added
+* Made `TextZones.Zone` type public
```

Comment:
These should be hyphens, not asterisks. Also please either add a fullstop at the end, or add a JIRA number.